File: app/src/main/java/app/simone/multiplayer/controller/KeysHandler.java
package app.simone.multiplayer.controller;

import java.util.ArrayList;

/**
 * Created by Giacomo on 02/08/2017.
 */

public class KeysHandler {

    private ArrayList<String> list;

    public KeysHandler(){
        list = new ArrayList<>();
    }

    public void addToList(String s){
        this.list.add(s);
    }

    public void clearList(){
        this.list.clear();
    }

    public String getElement(int index){
        return list.get(index);
    }

    public ArrayList<String> getList(){
        return this.list;
    }

}

File: app/src/main/java/app/simone/multiplayer/controller/KeysHandler.kt
package app.simone.multiplayer.controller

import java.util.ArrayList

/**
 * This class is just a container where match keys are stored while retrieving data from Firebase
 *
 * @author Giacomo
 */

class KeysHandler {

    var list: ArrayList<String> = ArrayList()

    fun addToList(s: String) {
        this.list.add(s)
    }

    fun getElement(index: Int): String = list[index]

}